### ISSUE TITLE
support wss://

### ISSUE THREAD

**Issue by nylira** [mappum/js-tendermint#10]
I'm trying to connect to the Tendermint RPC using WebSocket Secure:

```
let client = RpcClient("wss://nylira.net:443/websocket");
```

I get this error:

```
rpc.js?4b56:93 Uncaught (in promise) Error: SyntaxError: Failed to execute 'open' on 'XMLHttpRequest': Invalid URL
    at eval (rpc.js?4b56:93)
```

https://nylira.net is secured by TLS. I think this library doesn't support `wss` yet?

**Comment by mappum** [comment]
I think it should work if you remove "/websocket" from the URL. But I'll fix it to support both.

**Comment by nylira** [comment]
Thanks for the quick reply. I tried:

```
let client = RpcClient("wss://nylira.net/websocket");
let client = RpcClient("wss://nylira.net:443");
let client = RpcClient("wss://nylira.net");
``` 

And they all give me the same error.

```
let client = RpcClient("ws://nylira.net:80/websocket");
```

Is the one that works for localhost and HTTP domains, but is blocked on TLS-secured domains.

**Comment by mappum** [comment]
You were right, but this is now fixed in v3.1.10 😄 

You should now be able to connect with `let client = RpcClient("wss://nylira.net:443")`

**Comment by nylira** [comment]
Thank you! v3.1.10 fixes this.

But it seems v3.1.11 breaks it again
